ocs_ci/ocs/ocp.py

```diff
@@ -696,12 +696,10 @@ def get_resource(
         # WA, Failed to parse "oc get build" command
         # https://github.com/red-hat-storage/ocs-ci/issues/2312
         try:
-            if self.data['items'][0]['kind'].lower() == 'build':
-                value = (
-                    resource_info[column_index - 1] if len(titles) % 2 == 0
-                    else resource_info[column_index]
-                )
-                return value
+            if self.data['items'][0]['kind'].lower() == 'build' and (
```

Comment:
Any reason for removing the above code? 
AFAIK, @Akarsha-rai added it for fixing registry test failures.
May be we should handle this function in such a way that it satisfies both Jenkins and registry validations.
Please check.

Reply:
I added another condition so the change will only for Jenkins workload
`self.data['items'][0]['metadata']['annotations']['openshift.io/build-config.name'] == 'jax-rs-build'`